### PR TITLE
Add scripts to build Linux module wheels

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: scripts/internal/manylinux-build-module-wheels.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
```
Comment (jcfr) [inline scripts/internal/manylinux-build-module-wheels.sh]: I wonder if code with this script should be shared ? See https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/master/scripts/internal/manylinux-build-wheels.sh

File: scripts/internal/manylinux-build-module-wheels.sh [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+    echo ""
+    echo "PYTHON_EXECUTABLE:${PYTHON_EXECUTABLE}"
+    echo "PYTHON_INCLUDE_DIR:${PYTHON_INCLUDE_DIR}"
+    echo "PYTHON_LIBRARY:${PYTHON_LIBRARY}"
+
+    if [[ -e /work/requirements-dev.txt ]]; then
+      ${PYBIN}/pip install -r /work/requirements-dev.txt
+    fi
+    itk_build_dir=/work/ITK-$(basename $(dirname ${PYBIN}))-manylinux1_${arch}
```
Comment (jcfr) [inline scripts/internal/manylinux-build-module-wheels.sh]: That section seems to be the specific one

File: scripts/dockcross-manylinux-build-module-wheels.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```
Comment (jcfr) [inline scripts/dockcross-manylinux-build-module-wheels.sh]: May be usage of the script could be documented here ?

**Comment by thewtex** [comment]
Thanks for the reviews!

Good ideas.

I'll refactor to re-use common code in a follow-up PR. Also add more documentation. 

I think we should consider documenting with ReadTheDocs, too.

**Pull request merged**